fingerprint2.js

```diff
@@ -51,7 +51,27 @@
       }
       return target
     },
-    get: function (done) {
+    getOrDefaultPromise: function(defaultResult, defaultComponents){
```

Comment:
IMO `getOrDefaultPromise` this is too special-case to be included in the library, and the code is trivial, so I'd rather not include it. Do you think it should be included?

Reply:
I think it could be very common usecase so I include it. But I can move it into the README if you do not want it to be part of the library.

Reply:
Yes please!